.pre-commit-config.yaml

```diff
@@ -65,4 +65,4 @@ repos:
     rev: v2.5.0
     hooks:
       - id: prettier
-        files: ^js/.*$
```

Comment:
Do we not want to keep this check?

Reply:
I thought `types_or` appends to the existing checks. But I was mistaken. Thankfully, we can repeats hooks from the same repo as 4649f98.